tox.ini

```diff
@@ -74,6 +74,11 @@ commands =
 deps =
   {[testenv]deps}
   -r{toxinidir}/docs/requirements.in
+  # astroid is a dep for pylint, > 2.9.0 introduces false no-name-in-module and no-member errors
```

Comment:
Since it's an indirect dependency, please move it to the end of the list and state that it is transitive. I'd also add an entry line to make the distinction clear.